src/main/java/com/google/sps/data/AccessSecret.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.data;
+
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+import java.io.IOException;
+
+public class AccessSecret {
+  private String key;
+
+  public AccessSecret() throws IOException{
```

Comment:
perhaps instead of throwing the IOException all the way through, it should be captured and handled at some point before trying to set the key

Reply:
done